Sming/SmingCore/Network/WebsocketClient.cpp

```diff
@@ -99,10 +100,11 @@ void WebsocketClient::onError(err_t err)
 
 bool WebsocketClient::_verifyKey(char* buf, int size)
 {
-	String dd = String(buf);
-	uint8_t s = dd.indexOf("Sec-WebSocket-Accept: ");
+	String keyHeader("Sec-WebSocket-Accept: ");
+	String dd(buf);
+	uint8_t s = dd.indexOfIgnoreCase(keyHeader);
```

Comment:
Please keep in mind buf could be 1500bytes(MTU) packet You will create a new string  = heap + make tolower on it. I think this is not optimal that is why I created new search function to ignore case